src/main/webapp/partials/instructorHelpProfile.jsp

```diff
@@ -0,0 +1,209 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<h4>
+    <a name="profile">Profile</a>
+</h4>
+<div id="contentHolder">
+    <br>
+    <ol style="list-style-type: none;">
+        <li>
+            <span class="text-bold">
+                <a name="viewStudentProfile">
+                    <h3>1. Viewing student profiles</h3>
+                </a>
+            </span>
+            <div>
+                Go to the
+                <b>'Students'</b> page, click the panel heading for the relevant course. You will see a list of students for the course, then click
+                <b>'View'</b> button in the last column of the row corresponding to the student whose details you want to see.
+                <br>A new page should open which displays the student's profile, similar to the sample profile below -
+                <br><br>
+                <div class="row">
+                    <div class="col-md-2 col-xs-3 block-center">
+                        <img src="/images/profile_picture_default.png" class="profile-pic pull-right">
+                    </div>
+                    <div class="col-md-10 col-sm-9 col-xs-8">
+                        <table class="table table-striped">
+                            <thead>
+                                <tr>
+                                    <th colspan="2">Profile</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <tr>
+                                    <td class="text-bold">Short Name (Gender)</td>
+                                    <td>Alice
+                                        (<i> female </i>)
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td class="text-bold">Personal Email</td>
+                                    <td>alice@email.com</td>
+                                </tr>
+                                <tr>
+                                    <td class="text-bold">Institution</td>
+                                    <td>National University of Singapore</td>
+                                </tr>
+                                <tr>
+                                    <td class="text-bold">Nationality</td>
+                                    <td>American</td>
+                                </tr>
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+                <br>
+                Below that, is the student information table which displays course-related details of the student. It would appear something like this -
```

Comment:
Below that ~~,~~ is the student information table, which displays course-related **student** details ~~of the student~~. It **should look** ~~would appear~~ something like this: